index.html

```diff
@@ -28,7 +28,7 @@ <h2>Discuss</h2>
             <h2>Do</h2>
             <div class="list-group">
             <a href="/get-involved" class="list-group-item list-group-item-action">Get Involved</a>
-            <a href="/talks" class="list-group-item list-group-item-action">Talks</a>
+            <a href="/calendar" class="list-group-item list-group-item-action">Calendar</a>

```

Comment:
So does the calendar replace the Talks tab?

Reply:
On the home page list, not the navbar. The navbar is already really full so I opted to not add more to it. I think further linking/navbar changes would be best for another issue/PR

Reply:
the talks tab is still on the page's main navbar

Reply:
Yes, that was part of what I meant; this is intended behavior and not a bug